doc/oper.xml

```diff
@@ -1503,3 +1503,47 @@ true
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="AsSemigroup">
+  <ManSection>
+    <Func Name="AsSemigroup" Arg="filt, digraph"/>
```

Comment:
These should be documented as an operation, since it is an operation, as should IsMonoid